### PR TITLE
Fix kubetest race, simplify ouput()

### PR DESCRIPTION
**Pull request by fejta**
/assign @spxtr @krzyzacy 
Fixes https://github.com/kubernetes/test-infra/issues/3755

### CODE REVIEW

File: kubetest/util.go [hunk 1]
```diff
@@ -148,20 +150,27 @@ func finishRunning(cmd *exec.Cmd) error {
...
 			if err := syscall.Kill(-cmd.Process.Pid, syscall.SIGINT); err != nil {
-				log.Printf("Failed to interrupt %v. Will terminate immediately: %v", stepName, err)
+				log.Printf("Failed to interrupt %s. Will terminate immediately: %v", stepName, err)
 				syscall.Kill(-cmd.Process.Pid, syscall.SIGTERM)
 				cmd.Process.Kill()
 			}
 		case err := <-finished:
 			if err != nil {
-				return fmt.Errorf("error during %s: %v", stepName, err)
+				suffix := ""
```
Comment (spxtr) [inline kubetest/util.go]: Nit: prefer `var suffix string`.

**Comment by spxtr** [comment]
That's a wonderful number of added/removed lines.
/lint
/lgtm

File: kubetest/util.go (hunk 1 above)
Comment (fejta) [inline kubetest/util.go]: Done

**Comment by krzyzacy** [comment]
travis flake

**Pull request merged**